vcs/internal/path.go

```diff
@@ -0,0 +1,13 @@
+package internal
+
+import "strings"
+
+// Rel strips the leading "/" prefix from the path string, effectively turning
+// an absolute path into one relative to the root directory. A path that is just
+// "/" is treated specially, returning just ".".
+func Rel(path string) string {
+	if path == "/" {
+		return "."
+	}
```

Comment:
I've done some tests for `git` and `gitcmd`, and I found that this special handling of "/" input and returning "." was not necessary. Everything seems to work just fine if this if statement is removed.

@slimsag, if you can confirm that's the case for you as well, then I suggest let's remove this special handling. It will lead to simpler code and not doing something unnecessary.